migrations/versions/cb4ccda76a2b_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
+
+Revision ID: cb4ccda76a2b
+Revises: 0e80c49a6e28
+Create Date: 2019-06-03 22:57:01.500527
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = 'cb4ccda76a2b'
+down_revision = '0e80c49a6e28'
```

Comment:
@uds5501 change `down_revision` and `revises` to `Revision ID` of migration in my latest merged PR